index.src.html

```diff
@@ -548,6 +548,34 @@ <h1>Credential Management Level 1</h1>
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  <div algorithm="copy any buffer sources">
+    To <dfn>create a copy of any buffer sources' data</dfn> referenced by a value |V| of
+    [=dictionary=] type, run the following steps given |V|:
+
+    <ol class="algorithm">
+		1. Let |dictionaries| be a list consisting of |V| and all of |V|'s 
+			[=inherited dictionaries=], in order from least to most derived.
```

Comment:
Why do you need to walk through the inherited dictionaries? Doesn't |V| itself already have all of the members of dictionaries from which it derives?